src/SUMMARY.md

```diff
@@ -32,7 +32,7 @@
   * [Kafka](configuration/services/kafka.md)
   * [Memcached](configuration/services/memcached.md)
   * [MongoDB](configuration/services/mongodb.md)
-  * [MySQL](configuration/services/mysql.md)
+  * [MySQL/MariaDB](configuration/services/mysql.md)
```

Comment:
MariaDB is different from MySQL. Maybe put MariaDB then explain MariaDB is a fork of MySQL.

Reply:
From our POV as a hosting company they're 99% identical.  The differences are all application level, hence the proposal to combine the doc pages that would otherwise be 99% identical.